pipelines/nextflow/workflows/dumper_pipeline/main.nf

```diff
@@ -94,6 +105,19 @@ if (params.host && params.port && params.user && params.output_dir) {
     exit 1, "Missing server parameters"
 }
 
+if (params.select_dump) {
+    items = params.select_dump.split(/,/)
+    for (item in items) {
+        if (!default_selection.contains(item)) {
+            acceptable = default_selection.join(", ")
+            exit 1, "Selection item unknown: " + item + " (accepted: " + acceptable + ")"
+        }
+    }
+    params.selection = items
```

Comment:
I remember seeing something similar to pass multiple values for one argument in the BUSCO pipeline. Another good case to have some utils on some of this functionality (for now in our repo, later maybe in an Ensembl-wide one).